setup.py

```diff
@@ -45,7 +45,13 @@
 if sys.version_info < (3, 3):
     requirements.append("ipaddress")
 
-if platform.python_implementation() != "PyPy":
+if platform.python_implementation() == "PyPy":
+    if sys.pypy_version_info < (2, 6):
+        raise RuntimeError(
+            "cryptography 1.0 is not compatible with PyPy < 2.6. Please "
+            "upgrade PyPy or use an older version of cryptography."
```

Comment:
Drop the "older version of cryptography" bit. Dont' give people a wrong solution :-)